frontend/app/ui/profile/change-password.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+
+// layout
+import { Stack } from "@/app/ui/layout/stack";
+import { Button } from "@/components/ui/button";
+import { Card, CardContent, CardHeader } from "@/components/ui/card";
+import { Input } from "@/components/ui/input";
+
+export default function ChangePasswordForm() {
+  return (
+    <>
+      <Card className="w-80">
+        <CardHeader>Change Password</CardHeader>
+        <CardContent>
+          <Stack spacing={4}>
+            <Input
+              type="password"
+              id="current-password"
+              placeholder="Enter your current password"
+            />
+            <Input
+              type="password"
+              id="new-password"
+              placeholder="New Password"
+            />
+            <Input
+              type="password"
+              id="confirm-password"
+              placeholder="Confirm Password"
+            />
+            <Button>Submit</Button>
+          </Stack>
+        </CardContent>
+      </Card>
+    </>
+  );
+}
```

Comment:
I'll fix `6. The inputs do not have name attributes, which are typically used to identify form data after submission.`.
The rest is okay, because they are to be implemented later.